sdk/src/main/java/io/opentelemetry/sdk/resources/EnvVarResource.java

```diff
@@ -75,4 +78,35 @@ static Attributes parseResourceAttributes(@Nullable String rawEnvAttributes) {
       return attrBuilders.build();
     }
   }
+
+  /** Builder utility for this EnvVarResource. */
+  public static class Builder extends ConfigBuilder<Builder> {
+    private static final String OTEL_RESOURCE_ATTRIBUTES_ENV = "OTEL_RESOURCE_ATTRIBUTES";
+    private static final String OTEL_RESOURCE_ATTRIBUTES_KEY = "otel.resource.attributes";
+    private String envAttributes;
+
+    @Override
+    protected Builder fromConfigMap(
+        Map<String, String> configMap, NamingConvention namingConvention) {
+      String envAttributesValue = getStringProperty(OTEL_RESOURCE_ATTRIBUTES_ENV, configMap);
+      if (envAttributesValue != null) {
+        this.setEnvAttributes(envAttributesValue);
+      }
+
+      envAttributesValue = getStringProperty(OTEL_RESOURCE_ATTRIBUTES_KEY, configMap);
+      if (envAttributesValue != null) {
+        this.setEnvAttributes(envAttributesValue);
+      }
+      return this;
+    }
+
+    public String getEnvAttributes() {
```

Comment:
I'd suggest renaming this to `build` and doing the parsing and returning `Resource` here
```suggestion
    public Resource build() {
```